src/content_cao.cpp

```diff
@@ -614,7 +614,7 @@ GenericCAO::~GenericCAO()
 bool GenericCAO::getSelectionBox(aabb3f *toset) const
 {
 	if (!m_prop.is_visible || !m_is_visible || m_is_local_player
-			|| getParent() != NULL){
+			|| !m_prop.pointable || getParent() != nullptr) {
```

Comment:
`getParent()` does not return `nullptr`. Either revert this change to `NULL` or correct the function definitions and all lines where that function is checked against returning `NULL`. Never, ever mix these two.